doc-build/md040.md

```diff
@@ -29,6 +29,12 @@ You can configure the `allowed_languages` parameter to specify a list of
 languages code blocks could use. Languages are case sensitive. The default value
 is `[]` which means any language specifier is valid.
 
+You have the option to refuse extra data in fence code blocks information. To do
```

Comment:
I may have future comments on the phrasing here, but will wait until we agree on the parameter name.

Reply:
How about:

You can prevent extra data from being present in the info string of fenced code blocks. To do so, set the `language_only` parameter to `true`.
 <!-- markdownlint-disable-next-line no-space-in-code -->
 Info strings with leading/trailing whitespace (ex: `  js  `) or other content (ex: `ruby startline=3`) will trigger this rule.